install.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+
+# This is an install script for notes
+
+function assertInstalled() {
```

Comment:
Options here are to remove assertInstalled, and just assume we have everything, other option is to de-functionize it and put it inline. Thoughts?